src/transformers/models/auto/modeling_auto.py

```diff
@@ -29,7 +29,57 @@
 
 logger = logging.get_logger(__name__)
 
-
+MODEL_FOR_IMAGE_MAPPING_NAMES = OrderedDict(
```

Comment:
`MODEL_MAPPING_NAMES` should remain as the top mapping in this file. This object should probably just before other image related mappings, to keep consistency with the current (rough) ordering pattern